src/test/java/com/thebois/models/world/WorldTests.java

```diff
@@ -43,6 +46,35 @@ public void worldFind() {
         assertThat(worldObject).isEqualTo(null);
     }
 
+    @Test
+    public void getStructuresTest() {
+        // Arrange
+        final World world = new World(25);
+
+        // Act
+        world.createStructure(new Position(123, 123));
+        final IStructure structure = world.getStructures().get(0);
+
+        // Assert
+        assertThat(structure.getPosition()).isEqualTo(new Position(123, 123));
```

Comment:
To follow DRY, you could break out this `new Position(123, 123)` and the one on line 55 into a single constant that you set under `// Arrange`. This makes it easier to modify the test later on.